consensus/src/dag/dag_store.rs

```diff
@@ -226,46 +215,23 @@ impl Dag {
         self.nodes_by_digest.get(digest).cloned()
     }
 
-    pub fn get_unlinked_nodes_for_new_round(
+    pub fn get_strong_links_for_round(
         &self,
+        round: Round,
         validator_verifier: &ValidatorVerifier,
     ) -> Option<Vec<NodeMetadata>> {
```

Comment:
no strong opinion but I feel like Vec<NodeMetadata> should be a type called RoundParents.